src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -24,5 +24,35 @@ internal class FormattingContext
         public Range Range { get; set; }
 
         public Dictionary<int, IndentationContext> Indentations { get; } = new Dictionary<int, IndentationContext>();
+
+        public string GetIndentationLevelString(int indentationLevel)
+        {
+            var indentation = indentationLevel;
+            if (Options.InsertSpaces)
+            {
+                indentation *= (int)Options.TabSize;
+            }
+            var indentationString = GetIndentationString(indentation);
+            return indentationString;
+        }
+
+        public string GetIndentationString(int indentation)
```

Comment:
Is this method used outside here? If not, can make this private because it is confusing to have two methods.

Reply:
Ya it's used in the FormattingService